Documentation/ApiOverview/PasswordPolicies/Index.rst

```diff
@@ -37,7 +37,7 @@ The password policy applies to:
 *   Setting a new password for a backend user in :guilabel:`User settings`
 *   Resetting a password for a backend user
 *   Resetting a password for a frontend user (see also the feature toggle
-    ":ref:`typo3ConfVars_sys_features_security.usePasswordPolicyForFrontendUsers`")
+    ":confval:`security.usePasswordPolicyForFrontendUsers <t3coreapi:globals-typo3-conf-vars-sys-features-security-usepasswordpolicyforfrontendusers>`")
```

Comment:
You don't need the `t3coreapi:` prefix as it is the same manual.

Reply:
Done. Thanks for the hint ;)